lenskit/algorithms/als.py

```diff
@@ -465,14 +466,30 @@ def _train_iters(self, current, uctx, ictx):
             yield current
 
     def predict_for_user(self, user, items, ratings=None):
-        # look up user index
-        return self.score_by_ids(user, items)
+        scores = None
+        if ratings is not None:
+            u_offset = None
+            if self.bias:
+                ratings, u_offset = self.bias.transform_user(ratings)
+
+            ri_idxes = self.item_index_.get_indexer_for(ratings.index)
+            ri_good = ri_idxes >= 0
+            ri_it = ri_idxes[ri_good]
+            ri_val = ratings.values[ri_good]
+
+            u_feat = _train_bias_row_lu(ri_it, ri_val, self.item_features_, self.regularization)
+            print("user_features from predic_for_user: " + str(u_feat))
```

Comment:
Here we have another stray `print`. Either remote of change to `_log.debug`. I think maybe remove?